x-pack/filebeat/module/threatintel/misp/config/config.yml

```diff
@@ -34,12 +34,20 @@ response.split:
   target: body.response
   split:
     target: body.Event.Attribute
+    ignore_empty_value: true
     keep_parent: true
+    split:
+      target: body.Event.Object
+      keep_parent: true
+      split:
+        target: body.Event.Object.Attribute
```

Comment:
I think split supports multiple levels in a single split, can you confirm @marc-gr ? Or do we need to split on both Object and Attribute?

Reply:
Seems like the current situation is correct, feel free to ignore this.

Reply:
No, this does work if done in a single step. I've added this as a final commit here so we can easily revert is wanted.